.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,25 @@
+{
+	"name": "Hello",
+ 	"build": {
+		"dockerfile": "Dockerfile",
+		"args": {
+			"VARIANT": "bullseye"
+		}
+	},
+	"features": {
+		"ghcr.io/devcontainers/features/desktop-lite:1": {
+			"version": "latest"
+		}, "ghcr.io/devcontainers/features/docker-in-docker:2": {
+			"version": "latest"
+		}
+	},
+	"forwardPorts": [6080],
+	"postCreateCommand": "poetry install && source $(poetry env info --path)/bin/activate",
```

Comment:
I just did a clean checkout and open in a devcontainer and the post create command failed. Looks like it could not find source?
![image](https://github.com/IMAP-Science-Operations-Center/imap_processing/assets/6273429/104563b8-ebf7-4359-976c-a551c30221a9)

Reply:
@bryan-harter I also ran into this locally.

Reply:
```suggestion
	"postCreateCommand": "poetry install && . $(poetry env info --path)/bin/activate",
```